src/systempolicy/helperFunctions.go

```diff
@@ -152,3 +152,25 @@ func FilterSystemLogsByConfig(logs []types.KnoxSystemLog, pods []types.Pod) []ty
 
 	return filteredLogs
 }
+
+func GetWPFSSources() []string {
+	res, _, err := libs.GetWorkloadProcessFileSet(CfgDB, types.WorkloadProcessFileSet{})
+	if err != nil {
+		log.Error().Msgf("cudnot fetch WPFS err=%s", err.Error())
+		return nil
+	}
+
+	if res != nil {
```

Comment:
```suggestion
	if res == nil {
	    return nil
	}
```
This helps in reducing nesting.